docs/source/Ingestion/sprouter.md

```diff
@@ -35,44 +35,56 @@ Also, in the examples folder you will find `DegreesSeparation.scala` and `Tutori
 
 ## Local Deployment
 First lets open `TutorialRunner.scala`, which is our `main` class i.e. the file which we actually run. 
-To do this we have made our class a Scala app via `extends App`. 
-This is a short hand for creating your runnable main class (if you come from a Java background) or can be viewed as a script if you are more comfortable with Python. 
+To do this we have made our class a Scala app via `extends RaphtoryApp.Local`. Here, `Local` is an abstract class that implements the `main` method, making any class/object extending it "runnable". (This is a short hand for creating your runnable main class (if you come from a Java background) or can be viewed as a script if you are more comfortable with Python.) In adddition, it brings in local raphtory context in current scope. What it means is that overriding the `run` method from `RaphtoryApp` would provide you with a `RaphtoryContext` which creates (using `ctx.runWithNewGraph`) or returns already created graphs (`ctx.runWithGraph`) from the local machine itself. This is a pretty convenient way quickly try Raphtory.
```

Comment:
*In addition,

Reply:
Probably don't need bracket around "This is a short hand for creating your runnable ..."

Reply:
it brings in the local Raphtory context into the current scope. Overriding the `run` method from `RaphtoryApp` would provide you with a `RaphtoryContext` - `ctx.runWithNewGraph` creates a new graph and `ctx.runWithGraph` returns already created graphs from the local machine. This is a quick and convenient way to try Raphtory.

Reply:
Ignore this file!